tests/cypress/e2e/wp/form.cy.js

```diff
@@ -54,4 +54,112 @@ describe("MFP Work Plan E2E Submission", () => {
 
     cy.contains("Successfully Submitted").should("be.visible");
   });
+
+  it("Admin users can deny a Work Plan submission", () => {
+    cy.authenticate("adminUser");
+    cy.navigateToHomePage();
+    cy.get(
+      '[aria-label="List of states, including District of Columbia and Puerto Rico"]'
+    ).select("DC");
+    cy.get('[id="report-WP"]').click();
+    cy.contains("Go to Report Dashboard").click();
+    cy.wait(3000);
+
+    //There is a new submitted Workplan with the title
+    cy.contains(
+      `District of Columbia MFP Work Plan ${currentYear} - Period 1`,
+      { matchCase: true }
+    ).should("be.visible");
+
+    //Find our new program and open it
+    cy.get("table").within(() => {
+      cy.get("td")
+        .as("workplan")
+        .contains(
+          `District of Columbia MFP Work Plan ${currentYear} - Period 1`
+        );
+      cy.get("@workplan").parent().as("workplancontainer");
+      cy.get("@workplancontainer")
+        .find('button:contains("View")')
+        .as("viewButton");
```

Comment:
Theres a big debate about data-testids. Some say never use, or are a last resort and should be avoided. Some go all in. The argument against them is that they can cause brittle tests, particularly if there are multiple of the same test id on the page which this one is. Plus, there really isn't a good way to determine which one to access if we tried to index it to make it more unique. If a test fails at any point prior and then passes on the next attempt the number would be lost and would auto fail after. The reason this testId exists is we used to use them, and then went back through and removed almost all instances of it when Thomas Blackwell was on our team and with his recommendation. The reason its on the dashboard is because its a relic from MCR where we copied that code.